x/mint/keeper/genesis_test.go

```diff
@@ -111,17 +112,11 @@ func (suite *KeeperTestSuite) TestMintInitGenesis() {
 			originalVestingCoins := bankKeeper.GetBalance(ctx, developerAccount, tc.mintDenom)
 
 			// Test.
-			if tc.expectPanic {
-				suite.Panics(func() {
-					mintKeeper.InitGenesis(ctx, tc.mintGenesis)
-				})
+
+			if osmoutils.ConditionalPanic(suite.T(), tc.expectPanic || false, func() { mintKeeper.InitGenesis(ctx, tc.mintGenesis) }) {
```

Comment:
What is tc.expectPanic || false for?

Reply:
Also I recommend we change this to \
```suggestion
			osmoutils.ConditionalPanic(suite.T(), tc.expectPanic, func() { mintKeeper.InitGenesis(ctx, tc.mintGenesis) })
			if tc.expectPanic {
			    return
			}
```

Reply:
Oops i forgot that an unassigned bool is basically false